utils/find_missing_copyright_headers.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+expected_lines=(
+"/\*"
+" \* Copyright( \(C\))? 20[0-9][0-9](-20[0-9][0-9])? by the Widelands Development Team"
+" \*"
+" \* This program is free software; you can redistribute it and/or"
+" \* modify it under the terms of the GNU General Public License"
+" \* as published by the Free Software Foundation; either version 2"
+" \* of the License, or \(at your option\) any later version."
+" \*"
+" \* This program is distributed in the hope that it will be useful,"
+" \* but WITHOUT ANY WARRANTY; without even the implied warranty of"
+" \* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the"
+" \* GNU General Public License for more details."
+" \*"
+" \* You should have received a copy of the GNU General Public License"
+" \* along with this program; if not, write to the Free Software"
+"( \* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.)|( \* Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.)"
```

Comment:
```suggestion
" \* along with this program; if not, see <https://www.gnu.org/licenses/>."
```

This comes from the GPL3, but I think it's perfectly all right for GPL2 as well.
Although this means changing all files. Otherwise 51 Franklin street seems to be the current address, so at least files with 675 Mass Ave should be updated.